bundles/org.openhab.binding.connectedcar/README.md

```diff
@@ -0,0 +1,585 @@
+| range        | totalRange                  | Number:Length        | yes     | Total remaining range.                                                                          |
+|              | primaryRange                | Number:Length        | yes     | Range or the primary engine                                                                     |
+|              | secondaryRange              | Number:Length        | yes     | Range or the secondary engine                                                                   |
+|              | fuelPercentage              | Number:Dimensionless | yes     | Percentage of fuel remaining.                                                                   |
+|              | fuelMethod                  | String               | yes     | Method: 0=measured, 1=calculated                                                                |
+|              | gasPercentage               | Number:Dimensionless | yes     | Percentage of natural gas remaining                                                             |
+| charger      | chargingState               | String               | yes     | Current charging status                                                                         |
+|              | chargingStatus              | String               | yes     | Charging status                                                                                 |
+|              | chargingMode                | String               | yes     | Indicates the selected charging mode                                                            |
+|              | chargingLevel               | Number:Dimensionless | yes     | Current charging level in percent for an electrical car                                         |
+|              | batteryState                | Number               | yes     | Battery level                                                                                   |
+|              | remainingChargingTime       | Number:Time          | yes     | Time to reach a fully charged battery                                                           |
+|              | plugState                   | Switch               | yes     | State of the charging plug, ON=connected                                                        |
+|              | lockState                   | Switch               | yes     | ON: Plug is locked, OFF: Plug is unlocked and can be removed                                    |
+|              | powerState                  | String               | yes     | Indicates availability of charging power                                                        |
+|              | chargingPower               | Number:ElectricPoten | yes     | Current charging power                                                                          |
```

Comment:
The shortened version of the type looks weird in the rendered table. Better use the full name.